command/plugin_list.go

```diff
@@ -127,3 +115,15 @@ func (c *PluginListCommand) Run(args []string) int {
 		return OutputData(c.UI, res)
 	}
 }
+
+func (c *PluginListCommand) simpleResponse(plugins map[consts.PluginType][]string) []string {
+	out := []string{"Name | Type | Deprecation Status"}
```

Comment:
Quite a nice property of the previous code is that previous versions of clients can handle new fields being added from the server. Would it be possible to keep that, and if you want the formatting you've got here for deprecation status, add a special case for it or do some automated transforming to remove the underscore and capitalise the first letters etc?

Reply:
I've gone ahead and dropped this from the current PR. We will re-add in the next phase.